pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -123,6 +123,13 @@ type ClusterDeploymentSpec struct {
 	// PowerState defaults to the Running state.
 	// +optional
 	PowerState ClusterPowerState `json:"powerState,omitempty"`
+
+	// HibernateAfter will transition a cluster to hibernating power state after it has been running for the
+	// given duration. Running is assumed to be the time since the cluster was installed if no hibernating
+	// condition exists, otherwise the time since the cluster last transitioned that condition to false. If the cluster
```

Comment:
This may be a little too much detail for a user-facing comment. I suggest something like the following.

> The time that a cluster has been running is the time since the cluster was installed or the time since the cluster last came out of hibernation.